apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -225,3 +253,78 @@ function PopoverEndDate() {
     </p>
   `.toString();
 }
+
+function CreateCourseInstanceModal({
+  csrfToken,
+  initialStartDateFormatted,
+  initialEndDateFormatted,
+}: {
+  csrfToken: string;
+  initialStartDateFormatted?: string;
+  initialEndDateFormatted?: string;
+}) {
+  return Modal({
+    id: 'createCourseInstanceModal',
+    title: 'Create Course Instance',
+    formMethod: 'POST',
+    body: html`
+      <div class="form-group">
+        <label for="short_name">CIID (Course Instance Identifier)</label>
+        <input
+          type="text"
+          class="form-control"
+          id="short_name"
+          name="short_name"
+          required
+          pattern="[\\-A-Za-z0-9_\\/]+"
+          title="Please enter a valid format: only letters, numbers, hyphens, underscores, backslashes, and forward slashes."
+        />
+        <small class="form-text text-muted"
```

Comment:
You should use `aria-describedby` to link these bits of explanatory text with the input; this will help out folks who are using screen readers. There's an example of this on the email input in this example in the Bootstrap docs: https://getbootstrap.com/docs/5.3/forms/overview/#overview

Reply:
I added this to the inputs and the `small` text descriptions. Here's the updated short_name input, for example:

![image](https://github.com/user-attachments/assets/fae1d6a5-adf6-4072-bb55-31b1c6edc784)